index.bs

```diff
@@ -84,6 +159,14 @@ interface StorageBucket {
 };
 </xmp>
 
+A {{StorageBucket}} has a <dfn>bucket quota</dfn>, a number-or-null, initially null.
+Specifies the upper limit of usage which can be used by the bucket. It is not guaranteed
+that the specified quota will be available by the user agent.
+
+A {{StorageBucket}} has a <dfn>bucket expiration</dfn>, a timestamp-or-null, initially null.
+Specifies the upper limit of a bucket lifetime. The user agent MAY set a lower limit to enforce
```

Comment:
Is "lower limit" the right verbiage here? AFAIK there is not really a lower limit on duration (it could be evicted after 5 seconds in theory). I think some non-normative text that explains buckets can be evicted before their expiration date might be a good idea?

Reply:
lgtm